backend/api/app.py

```diff
@@ -110,7 +110,17 @@ def get_category(category: str, version: str) -> Response:
     return jsonify(get_category_mapping(category, get_categories_mapping(version)))
 
 
-@app.errorhandler(404)
+@app.route('/activity/installs/<plugin>')
+def get_plugin_installs(plugin: str) -> Response:
+    return jsonify(get_installs(plugin))
+
+
+@app.route('/activity/installs/<plugin>/stats')
+def get_plugin_installs_stats(plugin: str) -> Response:
+    return jsonify(get_installs_stats(plugin))
+
+
+@app.errorhandler(4034)
```

Comment:
```suggestion
@app.errorhandler(404)
```

Not sure what happened here lol but I have to assume it's a typo?